### PR TITLE
DCOS-41719: localize dates

### PR DESCRIPTION
**Pull request by natmegs**
Localize dates using DateFormat macro. Does not include relative dates.

Closes DCOS-41719.

## Testing

There should be no visual changes as a result of this PR.

## Trade-offs

Lingui `DateFormat` macro uses `Intl.DateTimeFormat` instead of `Date` functions like `toLocaleString` (not so much a tradeoff as just something to be aware of).

## Dependencies

None

## Screenshots

N/A

### CODE REVIEW

File: plugins/services/src/js/containers/pod-instances/__tests__/PodInstancesTable-test.js [hunk 1]
```diff
@@ -403,10 +425,25 @@ describe("PodInstancesTable", function() {
             return el.text().trim();
           });
 
+        const options = {
```
Comment (nLight) [inline plugins/services/src/js/containers/pod-instances/__tests__/PodInstancesTable-test.js]: Shall we extract those options? Maybe creating a wrapper component that would have them as a default?

File: plugins/nodes/src/js/components/LeaderGrid.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ import ConfigurationMapRow from "#SRC/js/components/ConfigurationMapRow";
 import ConfigurationMapSection from "#SRC/js/components/ConfigurationMapSection";
 import ConfigurationMapValue from "#SRC/js/components/ConfigurationMapValue";
 import Loader from "#SRC/js/components/Loader";
-import { msToRelativeTime, msToDateStr } from "#SRC/js/utils/DateUtil";
```
Comment (nLight) [inline plugins/nodes/src/js/components/LeaderGrid.js]: Shall we remove `msToDateStr` from `DateUtil` since we removed all the usages?

Comment (DanielMSchmidt) [reply plugins/nodes/src/js/components/LeaderGrid.js]: That would be so cool 😻

**Comment by natmegs** [comment]
Made a few changes to address feedback and I've removed the changes to the translation catalogs for now. They need to be done last so I will add once feedback has been finalized.

**Comment by GeorgiSTodorov** [comment]
I get this error
screenshot from 2018-12-03 11-48-11
and it is fixed if I change `import DateUtil from "#SRC/js/utils/DateUtil";` to `import DateUtil from "#SRC/js/utils/DateUtil.ts";`. Does anyone else have this problem?

**Comment by nLight** [comment]
@GeorgiSTodorov doesn't happen to me, hm..

**Comment by nLight** [comment]
@GeorgiSTodorov did you test with EE PR checked out?

**Comment by GeorgiSTodorov** [comment]
@nLight , good point, it works now, even without using the other PR. Not sure what broke it before.

**Pull request merged**